articles/quickstarts/search.md

```diff
@@ -0,0 +1,85 @@
+---
+title: "Q# quickstart: TODO"
+description: TODO
+author: cgranade
+ms.author: chgranad@microsoft.com
+ms.date: 10/9/2019
+ms.topic: article
+uid: microsoft.quantum.quickstarts.search
+---
+
+# Search for marked inputs with Grover's Algorithm
+
+This introduction uses [Grover's Algorithm][grover] to search inputs to a function for a particular _marked input_.
+This algorithm can be used to search for inputs with some special property, to find items in an unstructured list, and so forth.
+Grover's algorithm lets us use a quantum computer to run this search in less steps than the number of items in the list that we're searching — something no classical algorithm can do.
+
+In this introduction, we see how we can use Q# and the Quantum Development Kit to run Grover's algorithm with just a few lines of code.
+
+## Prerequisites
+
+- The Microsoft [Quantum Development Kit][install].
+
+## Writing Grover's Algorithm
+
+Using the Quantum Development Kit, create a new Q# project:
+
+```
+$ dotnet new console -lang Q# --output Grover
```

Comment:
Are we going to add instructions for how to do this in our other environments in tabs as well?

Reply:
I'm a bit worried about proliferating how many different copies of the project creation instructions we have kicking about, I guess.

Reply:
We (I :-)) could create a very short "how-to" guide called "How to create a Q# project", and we could link to it, as a pre-requisite ...